content/sensu-core/0.29/installation/installing-plugins.md

```diff
@@ -46,6 +49,15 @@ Successfully installed sensu-plugins-disk-checks-3.1.0
 1 gem installed
 [SENSU-INSTALL] successfully installed Sensu plugins: ["sensu-plugins-disk-checks"]{{< /highlight >}}
 
+### Pinning Versions
+
+We strongly recommend pinning the versions of any plugins installed in production.
```

Comment:
I think we need to do more here to describe what version pinning is. To me "version pinning" is a concept that involves both installing a specific version of a package, and using a mechanism to enforce that version. The sensu-plugin tool enables the former but does nothing to cover the latter requirement.

Reply:
That makes sense to me, we can add a bit about that in the main document about pinning and link here.

Reply:
@cwjohnston just to clarify what you mean in the case of os package managers such `apt`, `yum`, etc they have a mechanism to create a lock/hold which would prevent someone doing a generic update from installing it. In the case of ruby gem installs there is no such global guarantee though in your automated process be it `chef`, `puppet`, `ansible`, etc would ensure that the version to be installed (defined by strict or more loose constraints) will ensure that the version is installed but can't prevent someone with enough access from going to the CLI and manually installing it. The closest mechanism that exists for that is `bundler` but thats really for specific apps and other apps can install newer versions into a shared context if given access.